Lib/javascript/v8/std_map.i

```diff
@@ -61,6 +61,13 @@ namespace std {
                 std::map< K, T, C >::iterator i = self->find(key);
                 return i != self->end();
             }
+            std::vector<K> keys() {
+                std::vector<K> keys;
```

Comment:
I'd call `keys.reserve(self->size())` here, it could be a useful optimization for bigger maps.